### ISSUE TITLE
[Feature] Add event visibility indicator 

### ISSUE THREAD

**Issue by antontimmermans** [NPBruce/valkyrie#1451]
## Is your feature request related to a problem? Please describe.**
When you move events around (for example to add variation to a scenario), it happens that an event that once had text and an active button becomes a 'silent event'. (and sometimes the other way around). 
That event must become invisible (or silent).
This does not always happen, and it is difficult to spot.

## Describe the solution you'd like
Add a field + tickbox in the editor for **events** and **tokens** that shows if they are processed silently or not. In principle, it should who if the event or token contains `display=false` in its code'. With that tickbox, you should also be able to force it to be visible or not. 
 `display=false` : Box is empty

![image](https://user-images.githubusercontent.com/42072372/107120897-cdf61280-688f-11eb-8e73-1a119cb0c6d3.png)

**Comment by NicolaZee** [comment]
Good idea to show the Display Tick box.  That way it is easier to spot the bug has occurred.   But the underlying issue is a bug and sometime (hopefully) the bug will be fixed.  So, I suggest not having the tick box enabled and instead raise an issue to request the  bug to be fixed.
